pipes/WDL/workflows/demux_metag.wdl

```diff
@@ -27,7 +35,7 @@ workflow demux_metag {
 
   call metagenomics.krakenuniq as kraken {
     input:
-      reads_unmapped_bam = illumina_demux.raw_reads_unaligned_bams,
+      reads_unmapped_bam = dedup.dedup_bam
   }
   call reports.aggregate_metagenomics_reports as metag_summary_report {
```

Comment:
Can we call aggregate_metageomics_reports a second time on the kaiju outputs as well?

Reply:
`metagenomics.py::taxlevel_summary()` hasn't been adapted/tested to read kaiju summary files yet. I'd like that to be a separate PR (this one is already way beyond its initial scope).